eq_cir_management_ui/__init__.py

```diff
@@ -0,0 +1,134 @@
+
+
+def jinja_config(app: Flask) -> None:
+    """Configuration for the Flask Jinja2 component. Here we provide a custom loader,
+    so we can load from an array of sources.
+
+    :param app: The Flask application.
+    """
+    # loader for local templates and design system component templates
+    file_system_loader = FileSystemLoader([Path("./node_modules/@ons/design-system"), Path("./templates")])
+
+    app.jinja_loader = file_system_loader
+    app.jinja_env.undefined = ChainableUndefined
+    app.jinja_env.filters["env_override"] = env_override
+
+    # Clean up white space.
+    app.jinja_env.trim_blocks = True
+    app.jinja_env.lstrip_blocks = True
+
+
+def design_system_config() -> None:
+    """Set the version of the design system to an environment variable and add an
+    environment variable filter so environment variables can be read from within
+    Jinja. This enables the design system version to be defined once within the
+    package.json file and then reused throughout the application. Primarily to
+    declare the CSS version to use.
+    """
+    with open(Path("./package.json"), encoding="utf-8") as file:
+        package_json = json.load(file)
+
+        if "dependencies" in package_json:
+            try:
+                design_system_version = package_json["dependencies"]["@ons/design-system"]
+                if not Version.is_valid(design_system_version):
+                    logger.exception(
+                        "The '@ons/design-system' dependency version is invalid. "
+                        "Please ensure it follows semantic versioning.",
+                    )
+                    raise ValueError() from None
+
+                os.environ["DESIGN_SYSTEM_VERSION"] = design_system_version
+
+            except KeyError:
+                logger.exception(
+                    "The '@ons/design-system' dependency is not found in package.json. "
+                    "Please ensure it is listed under 'dependencies'.",
+                )
+                raise KeyError() from None
+
+
+def configure_secure_headers(app: Flask) -> None:
+    """Use Flask-Talisman to configure secure headers for the application.
+
+    :param app: The Flask application.
+    """
+    csp = {
+        "default-src": ["'self'", app.config["CDN_URL"]],
+        "font-src": ["'self'", app.config["CDN_URL"]],
+        "script-src": [
+            "'self'",
+            app.config["CDN_URL"],
+            "https://*.googletagmanager.com",
+            "https://*.google-analytics.com",
+        ],
+        "style-src": ["'self'", app.config["CDN_URL"]],
+        "connect-src": [
+            "'self'",
+            "https://*.googletagmanager.com",
+            "https://*.google-analytics.com",
+        ],
+        "frame-src": [],
+        "img-src": ["'self'", "data:", app.config["CDN_URL"]],
+        "object-src": ["'none'"],
+        "base-uri": ["'none'"],
+        "manifest-src": ["'self'", app.config["CDN_URL"]],
+    }
```

Comment:
To keep things tidier and as this is a const we should probably set the CSP outside of the method.

Reply:
As the CDN_URL is pulled in from the app this isn't really a constant.
I can see from the Runner setup that the the CSP is set as a constant, and then there are functions to inject in dynamic values. This feels like overkill, and makes getting a clear view of what is being set in the CSP that much harder.
As to make the change I'll have to implement in a similar way, am I missing the purpose of the Runner approach?